index.js

```diff
@@ -7,17 +7,15 @@ class ShoppingCart {
     this.total += price * quantity;
     if (this.items[itemName] !== undefined) {
       this.items[itemName] += quantity;
```

Comment:
Change all instances of the `[]` here to dot notation.

Reply:
i realized that not all [] can be changed to dot notation. It disrupt the test cases.

Reply:
Oh okay. Change the ones that can be changed then, and leave the others.

Reply:
Okay